packages/wrangler/src/config/validation.ts

```diff
@@ -1823,7 +1823,7 @@ const validateD1Binding: ValidatorFn = (diagnostics, field, value) => {
 	}
 	if (isValid && !process.env.NO_D1_WARNING) {
 		diagnostics.warnings.push(
-			`D1 Bindings are currently in alpha to allow the API to evolve before general availability.\nPlease report any issues to https://github.com/cloudflare/wrangler2/issues/new/choose\nNote: set NO_D1_WARNING=true to hide this message`
+			`D1 Bindings are currently in alpha to allow the API to evolve before general availability.\nPlease report any issues to https://github.com/cloudflare/wrangler2/issues/new/choose\nNote: set NO_D1_WARNING=true in your system's environment variables to hide this message`
```

Comment:
I'm not sure "system's environment variables" is any clearer tbh.
I know how to set a system level variable on Windows/Linux but not Mac. Really this should just be a command level one anyway but that's beside the point. 

Maybe it should just be like, "run the command with the environment variable NO_D1_WARNING=true like so: `NO_D1_WARNING=true npx wrangler d1 ..."

Though also just realised this is config validation so there probably isn't a way to get the current command... could do `npx wrangler ...` and hope that's clear enough but not a fan

I'm honestly not too sure how to make this understandable.

Reply:
Fair points